getting_started/build-guide/build-from-scratch.rst

```diff
@@ -261,6 +271,12 @@ from ``$SOF_WORKSPACE`` to them:
    cd "$SOF_WORKSPACE"
    for i in crosstool-ng/builds/xtensa-*; do ln -s "$i"; done
 
+Remove the temporarily build files (~7GB per toolchain)
+
+.. code-block:: bash
+
+   rm -rf .build
```

Comment:
Wow, great catch! Impressive. Why hide 7G in a hidden directory? Makes really no sense to me. gitignore is enough (I even tend to remove build directories from gitgnore locally. I digress)